lib/suma/admin_api/members.rb

```diff
@@ -60,6 +60,14 @@ class PreferencesEntity < BaseEntity
     expose :subscriptions, with: PreferencesSubscriptionEntity
   end
 
+  class DetailedOrganizationEntity < OrganizationEntity
+    expose :program_enrollments, as: :programs, with: Suma::AdminAPI::Entities::ProgramEnrollmentEntity
+  end
+
+  class DetailedOrganizationMembershipEntity < OrganizationMembershipEntity
+    expose :verified_organization, with: DetailedOrganizationEntity
```

Comment:
We don't expose detailed entities inside detailed entities. This sort of stuff should be setting off red flags.